backend/infrahub/graphql/mutations/node_getter/by_hfid.py

```diff
@@ -23,14 +23,56 @@ async def get_node(
         branch: Branch,
         at: str,
     ) -> Optional[Node]:
-        node = None
-        if not node_schema.human_friendly_id or "hfid" not in data:
-            return node
-
-        return await self.node_manager.get_one_by_hfid(
-            db=self.db,
-            hfid=data["hfid"],
-            kind=node_schema.kind,
-            branch=branch,
-            at=at,
-        )
+        if not node_schema.human_friendly_id:
+            return None
+
+        if "hfid" in data:
+            return await self.node_manager.get_one_by_hfid(
+                db=self.db,
+                hfid=data["hfid"],
+                kind=node_schema.kind,
+                branch=branch,
+                at=at,
+            )
+
+        for component in node_schema.human_friendly_id:
+            name = component.split("__")[0]
+            if name not in data.keys():
+                # The update neither includes "hfid" or all components to form an hfid:
+                return None
+
+        hfid: list[str] = []
+        for component in node_schema.human_friendly_id:
+            name, hfid_value = component.split("__", maxsplit=1)
+
+            if name in node_schema.attribute_names:
+                hfid_component = data.get(name).get(hfid_value)
+                if hfid_component is not None:
+                    hfid.append(hfid_component)
+            if name in node_schema.relationship_names:
+                relationship = [rel for rel in node_schema.relationships if rel.name == name][0]
+                related_node = await self.node_manager.find_object(
```

Comment:
I was hoping you could get multiple objects for a group of HFIDs with a single query like `get_many`, but it looks like we don't support that